src/twisted/conch/ssh/userauth.py

```diff
@@ -331,64 +349,67 @@ class SSHUserAuthClient(service.SSHService):
     making callbacks for more information when necessary.
 
     @ivar name: the name of this service: 'ssh-userauth'
-    @type name: L{str}
+
     @ivar preferredOrder: a list of authentication methods that should be used
         first, in order of preference, if supported by the server
-    @type preferredOrder: L{list}
+
     @ivar user: the name of the user to authenticate as
-    @type user: L{bytes}
+
     @ivar instance: the service to start after authentication has finished
-    @type instance: L{service.SSHService}
-    @ivar authenticatedWith: a list of strings of authentication methods we've tried
-    @type authenticatedWith: L{list} of L{bytes}
+
+    @ivar authenticatedWith: a list of strings of authentication methods we've
+        tried
+
     @ivar triedPublicKeys: a list of public key objects that we've tried to
         authenticate with
-    @type triedPublicKeys: L{list} of L{Key}
+
     @ivar lastPublicKey: the last public key object we've tried to authenticate
         with
-    @type lastPublicKey: L{Key}
     """
 
-    name = b"ssh-userauth"
-    preferredOrder = [b"publickey", b"password", b"keyboard-interactive"]
+    name: bytes = b"ssh-userauth"
+    preferredOrder: list[bytes] = [
+        b"publickey",
+        b"password",
+        b"keyboard-interactive",
+    ]
```

Comment:
Are these changes required by mypy ?
I was hoping that mypy is smart enougn and will automatically assing `bytes` as the type for `name`.

Reply:
Given the level of type confusion in this code around bytes vs. strings, I aesthetically prefer the over-annotating in this case; it was for me, not the type checker ;)